.gemini/skills/fix-mermaid/scripts/fix_mermaid.ts

```
@@ -0,0 +1,227 @@
+      result = fixHtmlMermaid(content);
+    } else if (ext === '.md' || ext === '.markdown') {
+      result = fixMarkdownMermaid(content);
+    } else if (['.tsx', '.ts', '.jsx', '.js', '.mjs', '.cjs'].includes(ext)) {
+      result = fixTsxMermaid(content);
+    } else {
+      console.error(`❌ Unsupported file type: ${ext}`);
+      process.exit(1);
+    }
+
+    if (result.report.length > 0) {
+      result.report.forEach(line => console.log(line));
```

Biome (2.4.16)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)